src/Team-Services-Bot.Api/Cards/AccountCard.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ———————————————————————————————
```

Comment:
It feels like there should be some tests for the cards. Can we test that the correct buttons have been added and there aren't any we don't expect? I will admit that this may not be such a "high value" test as some others.